app/components/Blockchain/Transaction/Transaction.jsx

```diff
@@ -48,12 +51,14 @@ export default class Transaction extends React.Component<Props> {
   render = () => {
     const {
       tx: { type },
+      chain,
       className,
       isPending,
     } = this.props
     return (
       <div className={classNames(styles.transactionContainer, className)}>
-        {this.renderAbstract(type)}
+        {chain !== 'neo3' && this.renderAbstract(type)}
+        {chain === 'neo3' && this.renderAbstractN3()}
```

Comment:
nitpick: Probably would be better to just have ternary here: `chain === 'neo3 ?  this.renderAbstractN3() : this.renderAbstract(type)`

Reply:
resolved in 
eac430b